expirationd.lua

```diff
@@ -252,8 +253,9 @@ local function get_task(name)
 end
 
 -- default process_expired_tuple function
-local function default_tuple_drop(space_id, args, tuple)
-    box.space[space_id]:delete(construct_key(space_id, tuple))
+-- task as parameter was added to preserve backward compatibility
+local function default_tuple_drop(space_id, args, tuple, task)
+    task.index:delete(construct_key(task.index, tuple))
```

Comment:
How will it work for non-unique indexes?

Reply:
Why isn't this change (in scope of this commit) covered with any tests?

Reply:
You're right, it won't work. Fixed, added a test in the commit scope